client/actions/loginActions.js

```diff
@@ -79,9 +67,7 @@ export function editProfile(body){
             .then((response) => {
                 dispatch(editProfileAction(response))
             })
-            .catch((err) => {
-                console.log("error ======>>>>>>", err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}